spec/step_spec.rb

```diff
@@ -6,7 +6,13 @@
         
         f = download_library("https://monitoring-sdk.firebaseapp.com/#{lib_version}/libTrace.a")
         expect(f.original_filename).to eq("libTrace.a")
-       
+    end
+
+    it 'returns zip file download is successful' do
+        lib_version = "latest"
+        
+        f = download_library("https://monitoring-sdk.firebaseapp.com/#{lib_version}/libTrace.a.zip")
+        expect(f.original_filename).to eq("libTrace.a.zip")
     end
 
     it 'raises error if url is 404' do
```

Comment:
Question: should this part be extended with the zip too? Is it possible for the expect to add an AND logic , something like this:
```
 expect {
            f = download_library("https://monitoring-sdk.firebaseapp.com/#{lib_version}/libTrace.a")
        }.to raise_error(Down::NotFound).and {
 f = download_library("https://monitoring-sdk.firebaseapp.com/#{lib_version}/libTrace.a.zip")
}
```

Not a ruby expert, so this is likely to be not correct 😅

Reply:
@richard-bogdan-bitrise added new tests for check zip content